app/assets/javascripts/application.js

```diff
@@ -10,7 +10,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
+= require jquery
```

Comment:
:+1: Didn't catch this. Surprised this doesn't break, but it definitely isn't loading.